frontend/components/landlord/LandlordInfo.tsx

```diff
@@ -49,8 +49,8 @@ const LandlordInfo = ({name, average, total}: IProps) => {
 				<p className="mt-1 text-sm text-gray-600">{t('landlord.share-sub')}</p>
 
 				<div>
-					<Link href="/create-review">
-						<p className="mt-2 inline-flex cursor-pointer items-center rounded-md border border-transparent bg-teal-600 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-teal-700 focus:outline-none focus:ring-2 focus:ring-teal-500 focus:ring-offset-2">
+					<Link  href="/create-review">
+						<p data-umami-event = "landlord page Submit a Review button" className="mt-2 inline-flex cursor-pointer items-center rounded-md border border-transparent bg-teal-600 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-teal-700 focus:outline-none focus:ring-2 focus:ring-teal-500 focus:ring-offset-2">
```

Comment:
Maybe we can create a different shorthand for the page name.

`landlord/${name}/submit-review` maybe. Have the other pages be similar.

Reply:
Sounds good. I am going to get all of the tests passing and then I can go back and change the naming conventions.